contrib/datawave-quickstart/bin/services/accumulo/install.sh

```diff
@@ -59,9 +59,9 @@ fi
 assertCreateDir "${DW_ACCUMULO_JVM_HEAPDUMP_DIR}"
 
 # Update tserver and other options in accumulo-env.sh
-sed -i "s~\(ACCUMULO_TSERVER_OPTS=\).*$~\1\"${DW_ACCUMULO_TSERVER_OPTS}\"~g" ${DW_ACCUMULO_CONF_DIR}/accumulo-env.sh
-sed -i "s~\(export JAVA_HOME=\).*$~\1\"${JAVA_HOME}\"~g" ${DW_ACCUMULO_CONF_DIR}/accumulo-env.sh
-sed -i "s~\(export ACCUMULO_MONITOR_OPTS=\).*$~\1\"\${POLICY} -Xmx2g -Xms512m\"~g" ${DW_ACCUMULO_CONF_DIR}/accumulo-env.sh
+sed -i'bak' "s~\(ACCUMULO_TSERVER_OPTS=\).*$~\1\"${DW_ACCUMULO_TSERVER_OPTS}\"~g" ${DW_ACCUMULO_CONF_DIR}/accumulo-env.sh
+sed -i'bak' "s~\(export JAVA_HOME=\).*$~\1\"${JAVA_HOME}\"~g" ${DW_ACCUMULO_CONF_DIR}/accumulo-env.sh
+sed -i'bak' "s~\(export ACCUMULO_MONITOR_OPTS=\).*$~\1\"\${POLICY} -Xmx2g -Xms512m\"~g" ${DW_ACCUMULO_CONF_DIR}/accumulo-env.sh
```

Comment:
Won't this leave backup files around that we need to clean up?

Reply:
I think you can do `sed -i''`

Reply:
Yup, that seems to work in my local testing. (Tested in a CentOS docker container too in order to make sure that doesn't break there.)

Reply:
Not an issue, really. Any cruft in DW_ACCUMULO_CONF_DIR will get obliterated automatically on an uninstall/reinstall, along with any other paths created by the quickstart.

Reply:
It would have been nice if `sed -i''` would have worked, but it didn't. I've added a `sed ... && rm ....bak` to the end of these to clean up the backup file if the sed completes successfully.

Reply:
I'll re-run and capture the specific error..

Reply:
Here's the bizarre error that I get regardless of whether I'm using a bare `-i` or `-i''`
```
sed: 1: "/Users/andrewfarris/pro ...": command a expects \ followed by text
```

Reply:
Ultimately, this works:
```
sed -i'' -e 's|.*\(export JAVA_HOME=\).*|\1'${JAVA_HOME}'|g' ${HADOOP_CONF_DIR}/hadoop-env.sh
```
Whereas this fails:
```
sed -i'' "s|.*\(export JAVA_HOME=\).*|\1${JAVA_HOME}|g" ${HADOOP_CONF_DIR}/hadoop-env.sh`
```